### DISCUSSION TITLE
【文件管理器】文件数量显示不一致

### DISCUSSION THREAD

**Discussion by lizipeng0013** [linuxdeepin/developer-center#4270] [Features Request & Ideas | 特性请求 & 头脑风暴]
![202305022020268482_录屏_dde-file-manager_20230502184819](https://user-images.githubusercontent.com/123794418/235836918-a111f74e-4eb6-441a-bdcb-1f7c1d419a88.gif)

在一个文件夹下选中所有文件和文件夹后点击属性，下方显示的“总文件数量”不包括文件夹，返回上一级后在这个文件夹上点属性后显示的“文件个数”又包括了文件夹

希望能统一

**Comment by BLumia** [comment]
cc @meiyixiang
